src/com/coding/patterns/dfs/MaxAreaOfIsland.java

```diff
@@ -0,0 +1,26 @@
+class Solution {
```

Comment:
Can you add leetcode problem link with complexity information? Follow other solutions structure.